packages/core/test/FieldTemplate_test.js

```diff
@@ -14,6 +14,21 @@ describe("FieldTemplate", () => {
     sandbox.restore();
   });
 
+  describe.only("FieldTemplate should only have one child", () => {
+    function FieldTemplate(props) {
+      if (React.Children.count(props.children) !== 1) {
+        throw "Got wrong number of children";
+      }
+      return null;
+    }
+    createFormComponent({
+      schema: { type: "string" },
+      uiSchema: { "ui:disabled": true },
+      FieldTemplate,
+    });
+    expect();
```

Comment:
Why do you have an empty `expect()` here?

Reply:
Removed.